Src/StdLib/Lib/mailcap.py

```diff
@@ -1,9 +1,16 @@
 """Mailcap file handling.  See RFC 1524."""
 
 import os
+import warnings
+import re
 
 __all__ = ["getcaps","findmatch"]
 
+_find_unsafe = re.compile(r'[^\xa1-\U0010FFFF\w@+=:,./-]').search
```

Comment:
This regex won't work with Python 2.

Reply:
Apologies, I missed that. I believe that should be resolved now with the updated regex.